content/en/blog/2024-05-13-announcing-flux-v2.3.0/index.md

```diff
@@ -0,0 +1,173 @@
+---
+author: Stefan Prodan
+date: 2024-05-13 15:00:00+00:00
+title: Announcing Flux 2.3 GA
+description: "We are thrilled to announce the release of Flux v2.3.0! Here you will find highlights of new features and improvements in this release."
+url: /blog/2024/05/flux-v2.3.0/
+tags: [announcement]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+We are thrilled to announce the release of [Flux v2.3.0](https://github.com/fluxcd/flux2/releases/tag/v2.3.0)!
+In this post, we will highlight some of the new features and improvements included in this release.
+
+## General availability of Flux Helm features and APIs
+
+This release marks a significant milestone for the Flux project, after almost four years of development,
+the helm-controller and the Helm related APIs have reached general availability.
+
+The following Kubernetes CRDs have been promoted to GA:
+
+- [HelmRelease](/flux/components/helm/helmreleases/) - `helm.toolkit.fluxcd.io/v2`
+- [HelmChart](/flux/components/source/helmcharts/) - `source.toolkit.fluxcd.io/v1`
+- [HelmRepository](/flux/components/source/helmrepositories/) - `source.toolkit.fluxcd.io/v1`
+
+The Helm features and APIs have been battle-tested by the community in production and are now considered stable.
+Future changes to the Helm APIs will be made in a backwards compatible manner,
+and we will continue to support and maintain them for the foreseeable future.
+
+### Enhanced Helm OCI support
```

Comment:
Also, it would be nice to mention the cli support for creating `helmChart` custom resources.

Reply:
Add to the CLI improvements section.